salt/modules/virt.py

```diff
@@ -3006,6 +3063,55 @@ def _set_vcpu(node, value):
             "set": lambda n, v: n.set("dev", v),
             "del": salt.utils.xmlutil.del_attribute("dev"),
         },
+        {
+            "path": "mem:hugepages",
+            "convert": _handle_unit,
+            "xpath": "memoryBacking/hugepages/page",
+            "get": _get_with_unit,
+            "set": _set_size_with_byte_unit,
+        },
+        {"path": "mem:nosharepages", "xpath": "memoryBacking/nosharepages"},
+        {"path": "mem:locked", "xpath": "memoryBacking/locked"},
+        {
+            "path": "mem:source",
+            "xpath": "memoryBacking/source",
+            "get": lambda n: str(n.get("type")) if n.get("type") else None,
+            "set": lambda n, v: n.set("type", str(v)),
+        },
+        {
+            "path": "mem:access",
+            "xpath": "memoryBacking/access",
+            "get": lambda n: str(n.get("mode")) if n.get("mode") else None,
+            "set": lambda n, v: n.set("mode", str(v)),
+        },
+        {
+            "path": "mem:allocation",
+            "xpath": "memoryBacking/allocation",
+            "get": lambda n: str(n.get("mode")) if n.get("mode") else None,
+            "set": lambda n, v: n.set("mode", str(v)),
+        },
+        {"path": "mem:discard", "xpath": "memoryBacking/discard"},
+        {"path": "mem:nosharepages", "xpath": "memoryBacking/nosharepages"},
+        {"path": "mem:locked", "xpath": "memoryBacking/locked"},
```

Comment:
I wonder what happens with those: they are not supposed to have text nor attributes